src/power-bi/Shared.Dataset/definition/tables/CostDetails.tmdl

```diff
@@ -1307,7 +1307,7 @@ table CostDetails
 		source = ```
 				let
 				    // Get the data
-				    RawData = ftk_Storage("focuscost"),
+				    RawData = ftk_Storage("focuscost", RangeStart, RangeEnd),
```

Comment:
If we pass this in, I believe we need to do it for the other datasets as well, right? Don't we want to do incremental refresh for all of them for consistency?

Reply:
Removed.